### ISSUE TITLE
Grunt option, issue with --option value set to  integer starting with 0

### ISSUE THREAD

**Issue by futur** [gruntjs/grunt-docs#173]
I'm running on _Mac 10.10.5_ with the following grunt installed,
```
grunt-cli v0.1.13
grunt v0.4.5
```

I have a task that will take a 3 digit number and it can be 000 or 099 or 002 or 999. Grunt option is converting the number starting with 0 to just number. 

> Example, 099 is returned as 99 by the grunt.option() function. One more example is 002 is returned as 2.

**Comment by JKAussieSkater** [comment]
Just to confirm:
* Grunt is removing "leading zeros" from your numbers
* You don't want this to happen?

Could you provide a minimal example of code which does this? At the moment, you haven't given us enough information to reproduce the problem or resolve it.

Might I suggest using strings instead? It's typical for number variables to have leading (or trailing) zeros removed because of optimisation and redundancy reasons, however string variables remain unaltered, so you could:
* Convert number to string
* If number is required again, convert string to number prior to calculation

**Comment by steveoh** [comment]
Is this a doc issue?

**Comment by AurelioDeRosa** [comment]
I agree with @steveoh. I don't think this is a doc issue and this should be reported on the main repository. To me this seems more an unexpected behavior. If the team decides that this is a "won't fix" and a documentation clarification is needed, then it can be addressed on a documentation level.

**Comment by JKAussieSkater** [comment]
Let's give @futur a week to respond with more details before we close this issue.

**Comment by futur** [comment]
@JKAussieSkater Thanks for the time. The following is the GruntFile I have and it is self explanatory.

```
module.exports = function(grunt) {

  // Project configuration.
  grunt.initConfig({
    pkg: grunt.file.readJSON('package.json'),
    prompt: {
    feature: {
      options: {
        questions: [
          {
            config: 'addFeature',
            type: 'input', 
            message: 'Enter the name of the feature. Input text and press enter.',
            default: 'testFeature', 
          }
        ],
        then:function(results,done){
          var target = grunt.option('target'); // leading zeros are getting removed right here
          // var target = grunt.option('target').toString(); // Doesnt work.
          // var target = grunt.option('target')+ ""; // Doesnt work too.
          grunt.log.ok("User input is : "+ results.addFeature); // This captures leading zero values OK values OK.
          grunt.log.ok("User target flag input is : "+ target); // This prints without leading zero.
        }
      }
    }
    }
  });

  // Load the plugin that provides the "uglify" task.
  grunt.loadNpmTasks('grunt-prompt');

  // Default task(s).
  grunt.registerTask('default', ['prompt:feature']);

};
```

**Comment by futur** [comment]
@steveoh @AurelioDeRosa Agreed. It is not a doc issue, this should be reported in the main repo. Its my mistake I put in the wrong place. I will put this in the main repo now. Please feel free to add this behaviour in your doc and close it this issue as well.

**Comment by JKAussieSkater** [comment]
>  [shama commented](https://github.com/gruntjs/grunt/issues/1474#issuecomment-189893994):
> Duplicate of [#1421](https://github.com/gruntjs/grunt/issues/1421) and fixed in `grunt@1.0.0-rc1`.

A part of me feels reluctant to document "resolved bugs" in the documentation, so I'll close this issue for now.
